### PR TITLE
added responsiveness to "remote provider" page

### LINKED ISSUE #10743: Responsiveness on 'Remote Provider' page.
### Current Behavior
 Remote provider capabilities & extensions were recently included in the Meshery UI's preferences, however, there is a responsiveness issue that needs fixing, primarily in mobile view. See attached image.

### Expected Behavior
 A responsive site that accounts for the mobile view of this new page. Here's a possible solution [in Figma]. (https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?type=design&node-id=29694%3A19485&mode=design&t=mElCikdb8NjkLBBo-1)

### Screenshots/Logs
What we have: 
Remote provider issue

What we could have: 
Figma remote provider page

### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by hargunkaur286**
**Notes for Reviewers**

This PR fixes #10743 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@nebula-aac here's one...

**Comment by vishalvivekm** [comment]
Thank you @hargunkaur286 
Let's discuss this in Meshery Dev meeting starting in 20 minutes.
Meeting minutes: https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#
Meeting link: https://meet.layer5.io/meshery

File: ui/components/UserPreferences/index.js [hunk 1]
```diff
@@ -142,11 +152,20 @@ const styles = (theme) => ({
   gridCapabilityHeader: {
     backgroundColor: theme.palette.type === 'dark' ? '#293B43' : '#7493A1',
   },
+  gridItem: {
+    backgroundColor: theme.palette.type === 'dark' ? '#3D4F57' : '#C9DBE3',
```
Comment (shubham251972) [inline ui/components/UserPreferences/index.js:156]: @hargunkaur286 thankyour for working on it, please use theme colors

**Review by shubham251972 — CHANGES_REQUESTED** [review]

File: ui/components/UserPreferences/index.js (hunk 1 above)
Comment (sudhanshutech) [inline ui/components/UserPreferences/index.js:156]: why adding these colors? Use themeing instead.

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Pull request closed without merging**